examples/fluids/navierstokes.c

```diff
@@ -965,6 +1007,17 @@ int main(int argc, char **argv) {
       bc.nwall = 0;
       bc.nslip[0] = bc.nslip[1] = bc.nslip[2] = 0;
     }
+    ierr = PetscOptionsIntArray("-bc_inflow",
+                              "Use inflow boundary conditions on this list of faces",
+                              NULL, bc.inflow,
+                              (len = sizeof(bc.inflow) / sizeof(bc.inflow[0]),
+                              &len), &flg); CHKERRQ(ierr);
+    if (flg) {
+      bc.ninflow = len;
+      // Using inflow boundaries disables automatic wall/slip boundaries (they must be set explicitly)
+      bc.nwall = 0;
+      bc.nslip[0] = bc.nslip[1] = bc.nslip[2] = 0;
+    }
```

Comment:
I think I need to remove these options since inflow and outflow BCs are invoked automatically by calling `-problem_advection_wind translation`.